helm/helm-2to3-migration-app/templates/job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.name }}
+      securityContext:
+        runAsUser: {{ .Values.userID }}
+        runAsGroup: {{ .Values.groupID }}
+      containers:
+        - name: {{ .Values.name }}
+          image: "{{ .Values.global.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+          command: ["/bin/sh" , "-c"]
+          args:
+            - for r in $(cat /data/config/releases | tr "," "\n"); do 2to3 convert --delete-v2-releases --tiller-ns {{ .Values.tiller.namespace }} "$r"; done;
```

Comment:
This is the critical part; looping through releases values, this chart would migrate all helm2 releases.

Reply:
This looks good. Just a question on the error handling. If one of the releases fails I guess the job will fail?

On the next execution as long as the list of un-migrated releases is updated we should be OK. But would be good to check that.

Reply:
Good points, I think it only restarts when the last execution failed. I will try to find out how we can restart it when it failed in the middle of the process.